src/Fractions/Fraction.Math.cs

```diff
@@ -269,8 +267,11 @@ namespace Fractions;
     /// </summary>
     /// <param name="fraction">The fraction.</param>
     /// <returns>The absolute value.</returns>
-    public static Fraction Abs(Fraction fraction) =>
-        new(BigInteger.Abs(fraction.Numerator), BigInteger.Abs(fraction.Denominator), fraction.State);
+    public static Fraction Abs(Fraction fraction) {
+        return fraction._normalizationNotApplied

```

Comment:
I don't understand why we would need this additional test. What speaks against the following?

```csharp
public static Fraction Abs(Fraction fraction) =>
        new(fraction._normalizationNotApplied, BigInteger.Abs(fraction.Numerator), BigInteger.Abs(fraction.Denominator));
```